vsintegration/src/FSharp.Editor/Options/EditorOptions.fs

```diff
@@ -33,7 +34,8 @@ type internal Settings [<ImportingConstructor>](store: SettingsStore) =
         
         store.RegisterDefault
             { ShowAfterCharIsTyped = true
-              ShowAfterCharIsDeleted = false }
+              ShowAfterCharIsDeleted = false
```

Comment:
I'm actually interested in seeing this be `true`, so that we behave the same as VS for Mac.  C# has this turned off by default for historical reasons that we don't share.